src/main/deploy/craneswerve/modules/backleft.json

```diff
@@ -0,0 +1,27 @@
+{
+  "location": {
+    "front": -8.5,
+    "left": 8.5
```

Comment:
Crane isn't a square, so these look like placeholders.

Reply:
Whoops, those should be modified